src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -219,6 +220,72 @@ assign(x = ".rs.acCompletionTypes",
                        excludeOtherCompletions = TRUE)
 })
 
+.rs.addFunction("attemptPlumberTagCompletion", function(token, line)
+{
+   emptyCompletions <- .rs.emptyCompletions(excludeOtherCompletions = TRUE)
+
+   # fix up tokenization
+   if (grepl("^\\s*#+\\*\\s*$", line) && token == "*")
```

Comment:
Nit, here and elsewhere: prefer `identical()` to compare strings in R since it won't do funny things if the LHS is something you don't expect (most commonly a vector of length != 1)